README.md

```diff
@@ -491,7 +491,7 @@ for details and use-cases.
 The following attributes are exported by the module:
 
 - **`user_pool`**: The `cognito_user_pool` object.
-- **`clients`**: A list of `cognito_user_pool_client` objects.
+- **`clients`**: A map of `cognito_user_pool_client` objects keyed by name'.
 - **`client_secrets`**: The secrets for all created `cognito_user_pool_client` resources.
```

Comment:
Let's maybe be even a bit more clear here as we are transforming the map before making it available as outputs:

```suggestion
- **`clients`**: A map of `cognito_user_pool_client` objects. The map is keyed by the `name` of the created clients. Client secrets are filtered out of this map and are available through the `client_secrets` output variable and flagged as sensitive.
- **`client_secrets`**: A sensitive map of client secrets for all created `cognito_user_pool_client` resources. The map is keyed by the `name` of the created clients.
```